src/components/OpenWith/OpenWith.js

```diff
@@ -61,6 +69,8 @@ class OpenWith extends PureComponent<Props, State> {
         extension: '',
         isDropdownOpen: false,
         integrations: null,
+        isLoading: true,
```

Comment:
Should this also be set to true in `componentDidUpdate` if the file changed? Or is it initial load only?

Reply:
yes, this should be reset whenever the file ID has changed.